sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2504,28 +2499,28 @@
           "type": "integer"
         },
         "deviceEtag": {
-          "description": "Device's ETag",
+          "description": "TODO - which operation does this affect?",
           "type": "string"
         },
         "status": {
-          "description": "Gets the corresponding Device's Status.",
+          "description": "The device's status.",
```

Comment:
Q - I have seen this before but just wondering what does these values "enabled"/"disabled" mean exactly for device status? 

Can we add something more to description for better readability?

Reply:
For a disabled device, it cannot connect to the service (and thus, cannot perform any operations).

Reply:
I assume, that's different than device being offline/not connected?

Reply:
Yes, device status is something you have control over, and can regulate using the IoT Hub control plane:
![image](https://user-images.githubusercontent.com/22563986/81840044-42687480-94fd-11ea-8cf9-e0fb4cf171e2.png)

Device offline/ not connected simply means that there isn't a device client instance available to acknowledge any communication that you send to the device.

Reply:
This is what we have for device status in the `Device` object, I'll use the same here as well:
`Flags whether a device is enabled or not. If disabled, a device cannot connect to the service.`